pytorch_impl/impute.py

```
@@ -0,0 +1,97 @@
+        sample = self.data[idx]
+        return torch.nn.functional.one_hot(sample, num_classes=self.depth).float()
+
+
+def load_model(path: str, device: torch.device) -> DeepImpute:
+    """Load a saved DeepImpute model."""
+    model = torch.load(path, map_location=device)
+    model.eval()
+    return model
+
+
+def impute_the_target(args: Dict) -> None:
```

Pylint (3.3.7)

[refactor] Too many local variables (21/15)

(R0914)